src/api_client.rs

```diff
@@ -339,16 +338,19 @@ impl Client {
     }
 
     /* config namespace */
-    pub async fn get_fork_schedule() -> Result<Vec<Fork>, Error> {
-        unimplemented!("")
+    pub async fn get_fork_schedule(&self) -> Result<Vec<Fork>, Error> {
+        let result: Value<Vec<Fork>> = self.get("eth/v1/config/fork_schedule/").await?;
+        Ok(result.data)
     }
 
-    pub async fn get_spec() -> Result<HashMap<String, String>, Error> {
-        unimplemented!("")
+    pub async fn get_spec(&self) -> Result<HashMap<String, String>, Error> {
+        let result: Value<HashMap<String, String>> = self.get("eth/v1/config/spec/").await?;
```

Comment:
could conceivably try for more precise typing than `HashMap<String, String>` here but after looking at the API docs this is easiest and just means the caller will have to do a bit more work but that seems ok